petastorm/tests/test_parquet_reader.py

```diff
@@ -220,3 +220,41 @@ def test_string_partition(reader_factory, tmpdir, partition_by):
         row_ids_batched = [row.id for row in reader]
     actual_row_ids = list(itertools.chain(*row_ids_batched))
     assert len(data) == len(actual_row_ids)
+
+
+def test_shuffle_per_row_group(scalar_dataset):
+    """Check if every row group is shuffled."""
+    for reader_factory in [make_batch_reader, make_reader]:
+        with reader_factory(scalar_dataset.url, reader_pool_type='dummy',
+                            shuffle_rows=True, shuffle_row_groups=False) as reader:
```

Comment:
Do you think we need to test that shuffle_rows=False work? My guess it's covered somewhere else, just wanted to make sure. Say we hardcode it to =True in make_reader/make_batch_reader: would other tests start failing?

Reply:
I think `shuffle_rows=False` is default case, which is tested in other existing reader tests. Those tests compare output order and length.